### ISSUE TITLE
Run Query keybinds don't work in tmux w/ SQLite db

### ISSUE THREAD

**Issue by muraii** [tconbeer/harlequin#425]
**Describe the bug**
When running `harlequin -a sqlite <dbname.db>` in a `tmux` session, neither `CTRL + ENTER` nor `CTRL + J` execute the query. Clicking the "run query" button works just fine, and both `CTRL + ENTER` and `CTRL + J` executes the query when running `harlequin` outside a `tmux` session on the same system.

**To Reproduce**
1. Create and attach new `tmux` session.
2. Execute `harlequin -a sqlite <dbname.db>`.
3. Write a valid SQL query.
4. Attempt to execute with `CTRL + ENTER` or `CTRL + J`.

**Expected behavior**
A valid SQL query is executed and results returned.

**Actual behavior**
No action appears to execute. There is no error or status message.

**Additional context**
What is the output of `harlequin --version`?
```
harlequin, version 1.12.0

Installed Adapters:
  - duckdb, version 1.12.0
  - sqlite, version 1.12.0
 ```

What database adapter are you using with Harlequin? (Default is DuckDB)
SQLite

Can you tell us more about your system?
- Shell: `zsh`
- Terminal: `Windows Terminal`
- OS of the shell: `Manjaro` in WSL2 on Windows 10
- OS of the terminal (if different from the shell): Windows 10

**Comment by tconbeer** [comment]
I can't reproduce in a slightly different environment (tmux, bash, ubuntu, wsl2 on win11).

What does your `tmux.conf` look like? Could you have bound those keys to something in tmux?

**Comment by muraii** [comment]
> What does your `tmux.conf` look like? Could you have bound those keys to something in tmux?

Ah yeah, I've adapted my `tmux.conf` to work with the `vim-tmux-navigator` plugin which looks for `CTRL-j` to be mapped to moving one pane down. I _knew_ there was some potential collision there but I forgot that not only do I need to specify that for the `nvim` configuration, I also did it in the `tmux` configuration.

Thing is, I don't have a configuration in `tmux.conf` nor in `vim-tmux-navigator.tmux` or related plugin stuff that captures `CTRL-ENTER`. I don't see any such configuration in my `zsh` dotfiles either, nor in the "Actions" list in Windows Terminal. But! I'll keep checking around.

Cheerio!
